lib/ui/profile/Profile.dart

```diff
@@ -184,15 +261,17 @@ class Profile extends StatelessWidget {
     );
   }
 
-  Widget _buildPage(
-      {BuildContext context, AsyncSnapshot<ProfileViewModel> snapshot}) {
+  Widget _buildPage({
+    BuildContext context,
+    AsyncSnapshot<ProfileViewModel> snapshot,
+  }) {
     final viewModel = snapshot.data;
     return SafeArea(
       child: SingleChildScrollView(
         child: Column(
           children: <Widget>[
             _buildHeader(context, viewModel),
-            _buildItemList(viewModel),
+            _buildItemList(viewModel, context),
```

Comment:
Maybe we can define this context as private attribute of class and then avoid to pass it on each function?

Reply:
Ok, then it's good to me